src/environments/gridworlds.jl

```diff
@@ -0,0 +1,26 @@
+export RLBaseGridWorld
+import .GridWorlds
+
+mutable struct RLBaseGridWorld{T<:GridWorlds.AbstractGridWorld} <: RLBase.AbstractEnv
+    env::T
+end
+
+Base.convert(::Type{RLBase.AbstractEnv}, env::GridWorlds.AbstractGridWorld) = convert(RLBaseGridWorld, env)
+
+Base.convert(::Type{RLBaseGridWorld}, env::GridWorlds.AbstractGridWorld) = RLBaseGridWorld(env)
+
+RLBase.get_state(env::RLBaseGridWorld) = GridWorlds.get_state(env.env)
+
+RLBase.get_actions(env::RLBaseGridWorld) = GridWorlds.get_actions(env.env)
+
+RLBase.get_reward(env::RLBaseGridWorld) = GridWorlds.get_reward(env.env)
+
+RLBase.get_terminal(env::RLBaseGridWorld) = GridWorlds.get_terminal(env.env)
```

Comment:
Usually the function name shouldn't be tied to a specific package. Here the `get_terminal` is not straight forward in the `GridWorlds` package. I'd suggest a name of `is_mission_completed` or simply `is_done` in GridWorlds.